src/main/resources/peer-review-transform.xsl

```diff
@@ -189,4 +200,9 @@
   </xsl:template>
   
   <xsl:template match="named-content" />
+
+  <!-- ignore received date found during template processing.
+       date is referenced directly in revision #0 block --> 
+  <xsl:template match="received-date" />
```

Comment:
re: dev note 2:
I think this sort of no-op is a common pattern where you want to present the value of something in a different location but you don't want the value to appear again in it's original position due to it passing through via `<apply-templates>`.  My XSL experience is limited but I don't consider it a smell.

These no-ops might even make up the majority of the article transform XSL.

Reply:
ah. ok. I really don't have that much experience with xsl. let me rephase. smell is actually a pleasant aroma.